README.md

```diff
@@ -268,6 +268,36 @@ requires an above average amount of memory available. To avoid JS Heap overflows
 npm scripts that run webpack (`start`, `start:dev`, and `start:poll`) pass an argument
 `--max_old_space_size` to increase the memory allocation. This can be adjusted as necessary.
 
+## Release Checklist
+The following conditions must be tested before deploying a release. If any of the following
+circumstances causes a break, CHANGELOG should indicate a workaround or migration steps
+should be provided.
```

Comment:
Maybe don't need "should be provided"? Just "...indicate a workaround or migration steps"